mlir/include/mlir/Dialect/MIOpen/Tuning/ConvContext.h

```diff
@@ -205,5 +226,42 @@ template <typename T> static ConvolutionContext populateConvContext(T &op) {
           dilationVal, paddingVal, gemmId, dataType};
 }
 
+#define POPULATE_CONV_CONTEXT(T) \
```

Comment:
Instead of using macro, why not use template as-is, since you are templatized on the op itself?

Reply:
The existing template assumes `T` be `GridwiseGemmOp` or `GridwiseGemmV2Op` which doesn't have `filter` / `input` / `output` fields, so I have to resort to specialized templates.

This will be changed in the next iteration.